falcon_multiqc/commands/save.py

```diff
@@ -148,8 +181,33 @@ def cli(directory, sample_metadata, batch_description, cohort_description, batch
             for line in batch_metadata:
                 split = line.split(",")
                 batch_name = split[0]
+                number_of_samples = split[1] # to be implmented (have to modify models)
                 batch_description = split[2]
                 # Update each batch with the given batch description.
-                (session.query(Batch)
-                    .filter(Batch.batch_name == batch_name, Batch.cohort_id == cohort_id)
-                    .one().description) = batch_description
+                try:
+                    (session.query(Batch).filter(Batch.batch_name == batch_name).one().description) = batch_description
+                    session.commit()
+                    click.echo(f"Batch '{batch_name}' description has been saved.")
+                except NoResultFound:
+                    click.echo(f"Batch '{batch_name}' is not present in the database so description cannot be added, exiting.")
+                    sys.exit(1)
+
+        # Save Cohort metadata.
+        if cohort_metadata:
+            # Skip header
+            next(cohort_metadata)
+            for line in cohort_metadata:
+                split = line.split(",")
+                cohort_name = split[0] # to be implmented (have to modify models)
+                number_of_samples = split[1] # to be implmented (have to modify models)
+                Number_of_batches = split[2] # to be implmented (have to modify models)
+                type = split[3] # to be implmented (have to modify models)
```

Comment:
I think again remove. Didn't put it in purposely because this is already in the sample metadata. Unless it's different and we should double check with kaplan team.

Reply:
Yeah, but maybe we should consider concatenating all those other fields into the description?>

Reply:
# of samples and # of batches we can get from just SQL / counting, unless they have a specific reason of wanting a separate field for that.

We should ask them if we can leave it out or they want them for some reason - will ask now

Reply:
hmm yeah true haha